### ISSUE TITLE
App is crashing on Android 8.1

### ISSUE THREAD

**Issue by andreschab90** [stanwood/Bitrise_Android#30]
The client is immediately crashing after login with valid credentials. Environment: `android 8.1`

**Comment by ubuntudroid** [comment]
Hey, @andreschab90! Thanks for reporting. Can you provide us with a log? We know you can. ;)

**Comment by andreschab90** [comment]
Hey @ubuntudroid, sure i can 😆 

```
03-02 22:55:55.555 18378-18378/? E/AndroidRuntime: FATAL EXCEPTION: main
                                                   Process: io.stanwood.bitrise, PID: 18378
                                                   java.lang.NullPointerException: Attempt to invoke virtual method 'android.graphics.drawable.Drawable io.stanwood.bitrise.data.model.Platform.getIcon(android.content.res.Resources)' on a null object reference
                                                       at io.stanwood.bitrise.ui.dashboard.vm.AppViewModel.getIcon(AppViewModel.kt:36)
                                                       at io.stanwood.bitrise.databinding.ItemAppBinding.executeBindings(ItemAppBinding.java:178)
                                                       at android.databinding.ViewDataBinding.executeBindingsInternal(ViewDataBinding.java:430)
                                                       at android.databinding.ViewDataBinding.executePendingBindings(ViewDataBinding.java:402)
                                                       at io.stanwood.bitrise.util.databinding.ViewModelAdapter$ViewHolder.bind(ViewModelAdapter.java:58)
                                                       at io.stanwood.bitrise.util.databinding.ViewModelAdapter.onBindViewHolder(ViewModelAdapter.java:95)
                                                       at io.stanwood.bitrise.util.databinding.ViewModelAdapter.onBindViewHolder(ViewModelAdapter.java:17)
                                                       at android.support.v7.widget.RecyclerView$Adapter.onBindViewHolder(RecyclerView.java:6508)
                                                       at android.support.v7.widget.RecyclerView$Adapter.bindViewHolder(RecyclerView.java:6541)
                                                       at android.support.v7.widget.RecyclerView$Recycler.tryBindViewHolderByDeadline(RecyclerView.java:5484)
                                                       at android.support.v7.widget.RecyclerView$Recycler.tryGetViewHolderForPositionByDeadline(RecyclerView.java:5750)
                                                       at android.support.v7.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:5589)
                                                       at android.support.v7.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:5585)
                                                       at android.support.v7.widget.LinearLayoutManager$LayoutState.next(LinearLayoutManager.java:2231)
                                                       at android.support.v7.widget.LinearLayoutManager.layoutChunk(LinearLayoutManager.java:1558)
                                                       at android.support.v7.widget.LinearLayoutManager.fill(LinearLayoutManager.java:1518)
                                                       at android.support.v7.widget.LinearLayoutManager.onLayoutChildren(LinearLayoutManager.java:610)
                                                       at android.support.v7.widget.RecyclerView.dispatchLayoutStep2(RecyclerView.java:3719)
                                                       at android.support.v7.widget.RecyclerView.dispatchLayout(RecyclerView.java:3436)
                                                       at android.support.v7.widget.RecyclerView.onLayout(RecyclerView.java:3988)
                                                       at android.view.View.layout(View.java:19659)
                                                       at android.view.ViewGroup.layout(ViewGroup.java:6075)
                                                       at android.support.v4.widget.SwipeRefreshLayout.onLayout(SwipeRefreshLayout.java:611)
                                                       at android.view.View.layout(View.java:19659)
                                                       at android.view.ViewGroup.layout(ViewGroup.java:6075)
                                                       at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1791)
                                                       at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1635)
                                                       at android.widget.LinearLayout.onLayout(LinearLayout.java:1544)
                                                       at android.view.View.layout(View.java:19659)
                                                       at android.view.ViewGroup.layout(ViewGroup.java:6075)
                                                       at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
                                                       at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
                                                       at android.view.View.layout(View.java:19659)
                                                       at android.view.ViewGroup.layout(ViewGroup.java:6075)
                                                       at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
                                                       at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
                                                       at android.view.View.layout(View.java:19659)
                                                       at android.view.ViewGroup.layout(ViewGroup.java:6075)
                                                       at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1791)
                                                       at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1635)
                                                       at android.widget.LinearLayout.onLayout(LinearLayout.java:1544)
                                                       at android.view.View.layout(View.java:19659)
                                                       at android.view.ViewGroup.layout(ViewGroup.java:6075)
                                                       at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
                                                       at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
                                                       at android.view.View.layout(View.java:19659)
                                                       at android.view.ViewGroup.layout(ViewGroup.java:6075)
                                                       at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1791)
                                                       at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1635)
                                                       at android.widget.LinearLayout.onLayout(LinearLayout.java:1544)
                                                       at android.view.View.layout(View.java:19659)
03-02 22:55:55.555 18378-18378/? E/AndroidRuntime:     at android.view.ViewGroup.layout(ViewGroup.java:6075)
                                                       at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
                                                       at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
                                                       at com.android.internal.policy.DecorView.onLayout(DecorView.java:761)
                                                       at android.view.View.layout(View.java:19659)
                                                       at android.view.ViewGroup.layout(ViewGroup.java:6075)
                                                       at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:2496)
                                                       at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2212)
                                                       at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1392)
                                                       at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:6752)
                                                       at android.view.Choreographer$CallbackRecord.run(Choreographer.java:911)
                                                       at android.view.Choreographer.doCallbacks(Choreographer.java:723)
                                                       at android.view.Choreographer.doFrame(Choreographer.java:658)
                                                       at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:897)
                                                       at android.os.Handler.handleCallback(Handler.java:790)
                                                       at android.os.Handler.dispatchMessage(Handler.java:99)
                                                       at android.os.Looper.loop(Looper.java:164)
                                                       at android.app.ActivityThread.main(ActivityThread.java:6494)
                                                       at java.lang.reflect.Method.invoke(Native Method)
                                                       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
                                                       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
```

**Comment by mmoczkowski** [comment]
@andreschab90 Many thanks for giving us this feedback <3 Hopefully it's fixed now with the 1.2 release!
